$OpenBSD: patch-daemon_gdm-session-record_c,v 1.1 2011/09/12 07:39:21 ajacoutot Exp $

implicit declaration of function 'logwtmp'
implicit declaration of function 'login'

--- daemon/gdm-session-record.c.orig	Fri Jun 17 15:02:07 2011
+++ daemon/gdm-session-record.c	Fri Jun 17 15:02:27 2011
@@ -33,6 +33,8 @@
 #include <utmp.h>
 #endif
 
+#include <util.h>
+
 #include <glib.h>
 #include <glib/gi18n.h>
 #include <glib/gstdio.h>
